src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
```

Comment:
Avoid using verbs like 'Drawable' and 'AreaCalculator' for interface names. A better naming convention would be 'Drawable' and 'AreaCalculable' or similar noun-based names that describe the capability.